features/scala-insert-import.feature

```diff
@@ -132,6 +132,32 @@ Feature: Insert Scala Import
     }
     """
 
+  Scenario: Insert in File without package
+    When I open temp scala file "test"
+    And I insert:
+    """
+    import scala.concurrent.duration._
+    class A(value:String){
+      def hello(){
+        println(new /*1*/ListBuffer())
```

Comment:
probably your line could just be `new ListBuffer` instead of wrapping with `println` and brackets, to better simulate the usecase.